.github/workflows/release-cli.yaml

```diff
@@ -24,10 +24,10 @@ jobs:
     steps:
     - name: Fetch required Details
       run: |
-          touch latestRelease.json && curl https://api.github.com/repos/apicurio/apicurio-registry/releases/latest > latestRelease.json
-          echo "RELEASE_VERSION=$(cat latestRelease.json | jq -r '.name')" >> $GITHUB_ENV
-          echo "SOURCE_CODE_URL=$(cat latestRelease.json | jq -r '.zipball_url')" >> $GITHUB_ENV
-          echo "UPLOAD_URL=$(cat latestRelease.json | jq -r '.upload_url')" >> $GITHUB_ENV
+        touch latestRelease.json && curl https://api.github.com/repos/${GITHUB_REPOSITORY}/releases | jq '.[0]' > latestRelease.json
```

Comment:
`releases/latest` endpoint does not fetch pre-releases. Hence, hitting `/releases` endpoint and grabbing the first element of the array for the most-recent release.

Also, using the ${GITHUB_REPOSITORY} variable provided by GitHub Actions.